distance.go

```diff
@@ -0,0 +1,4675 @@
+package tlsh
+
+// bitPairsDiffTable is pre-calculated table that represents approximation 
```

Comment:
Oh, _I_ can find it :) I just like to make it easier for others to figure it where your magic numbers come from.